packages/core/src/index.ts

```diff
@@ -214,10 +214,16 @@ export class App {
    * While singleton classes are typically side effect free and can be instantiated lazily when
    * first requested, plugins have side-effects, such as adding router routes, adding RPC endpoints
    * or setting up event listeners. The load method is therefore used to load those plugins.
+   *
+   * You can load a plugin only once; load throws an error the second time.
    */
   load<T>(Klass: ConstructorOrFactory<App, T>): void {
-    this.getSingleton(Klass); // force initialization;
-    return;
+    if (this.isSingletonProvided(Klass)) {
```

Comment:
In this case it should be just getSingleton (multiple loads should be ok)

Reply:
I think we need to re-do this with a bit of thinking.

Reply:
Redo this PR or the loading-once-or-twice mechanics? Fine with both really.

Reply:
We need to think whether we want to allow loading plugins within plugins, because thats what implies multiple load calls are OK.